internal/solitaire/solitaire.go

```diff
@@ -179,6 +185,15 @@ func (s *Solitaire) move(to *index) bool {
 		if s.selected.card == fromDeck.Size()-1 && toDeck.Size() == 0 && fromDeck.Top().Value == 0 || toDeck.Size() > 0 && fromDeck.Top().Value == toDeck.Top().Value+1 && fromDeck.Top().Suit == toDeck.Top().Suit {
 			s.toggleSelect(s.selected)
 			toDeck.Add(fromDeck.Pop())
+      card := toDeck.Top().String()
+      cardArr := strings.Split(card, "")
```

Comment:
I don't think we need to do this, `card[0]` should give us the sam value as `cardArr[0]`, right?

Reply:
It is but i didn't want to convert it to a char. But yes it would be better to do it that way.

Reply:
This was meant for my personal use because i wanted to get a victory message, i thought i'll try sending you a pull request if you maybe wanted to add it in.